components/automate-cli/pkg/diagnostics/integration/iam_v2_policies.go

```diff
@@ -44,14 +44,15 @@ type PolicyParameters struct {
 	ProjectID string
 }
 
-type iamV2PoliciesSave struct {
+type generatedV2PolicyData struct {
```

Comment:
TODO: change the other diagnostics to be consistent with this naming scheme

Reply:
TODO before merge?